<!---->
<launch>
	<node name="dynamixel_mediator" pkg="hostess_skeleton_tracker" type="dynamixel_mediator" output="screen">
		<param name="DeviceIndex" type="int" value="4"/>
		<param name="Baudnum" type="int" value="1"/>
		<param name="UpdateRate" type="int" value="20"/>
		<param name="YamlPath" type="string" value="$(find hostess_skeleton_tracker)/launch/include/pan_joint.yaml"/>
	</node>
</launch>

